sdk/identity/azure-identity/CMakeLists.txt

```diff
@@ -63,6 +77,7 @@ target_include_directories(
   azure-identity
     PUBLIC
       $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc>
+      $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/build/sdk/core/azure-core/inc>
```

Comment:
What happen when `build/sdk/core/azure-core/inc>` does not exists because we are using VCPKG or building from root?  Will cmake just ignore this?

Reply:
has no issue , it's a look up path,